bluepill/src/BPPacker.m

```diff
@@ -161,8 +198,37 @@ @implementation BPPacker
         } else {
             return NSOrderedSame;
         }
-    }]];
+    }];
+    [BPUtils printInfo:INFO withString:@"Here are splitted and sorted bundles..."];
```

Comment:
This sentence reads a little weird, I'm not sure what it's trying to say.

Reply:
Fixed it and hopefully made it more descriptive.